### DISCUSSION TITLE
Previous version of Sketch

### DISCUSSION THREAD

**Discussion by borodean** [Homebrew/discussions#608] [Casks] [answered]
Figuring out how to install one of the previous versions of Sketch.

Use case: Sketch sells one-year licenses, and I want to install the latest version for which my license is valid. Technically it is possible, because all versions continue to be available for download (see the URL in https://github.com/Homebrew/homebrew-cask/blob/master/Casks/sketch.rb).

The old-fashioned method of `brew cask install https://raw.githubusercontent.com/Homebrew/homebrew-cask/<commit>/Casks/sketch.rb` no longer works.

I was unable to create a private cask with `brew extract`. As far as I understand, it only works with regular formulas.

The `homebrew/cask-versions` tap does not contain specific versions, just `sketch-beta`.

What is the recommended approach in this situation?

**Comment by SMillerDev** [comment, marked as answer]
Create your own tap and put the old cask in the tap using:
```sh
mkdir Casks
curl -o Casks/sketch@version.rb https://raw.githubusercontent.com/Homebrew/homebrew-cask/<commit>/Casks/sketch.rb
```
